userguide/sound_classifier/how-it-works.md

```diff
@@ -0,0 +1,52 @@
+# How Does this Work?
+
+Training and making predctions, for a sound classifier model, is a three
```

Comment:
Typo: predictions, not predctions.

Nit: I don't think there should be a comma between "predictions" and "for"? Not sure...

Reply:
Done and done.